src/main/java/by/exadel/internship/service/impl/InterviewServiceImpl.java

```diff
@@ -57,21 +61,41 @@
                 throw new InappropriateRoleException("Inappropriate role", "user.role.invalid");
             }
         }
+
+        return getAllInterviewWithUser(interviews);
+    }
+
+    public List<InterviewWithUserNameDTO> getAllInterviewWithUser(List<Interview> interviews){
+        MDCLog.putClassNameInMDC(SIMPLE_CLASS_NAME);
         List<InterviewWithUserNameDTO> interviewWithUserNameDTOList =
                 mapper.mapToInterviewWithUserDTO(interviews);
         interviewWithUserNameDTOList.forEach(interview -> {
             interview.setAdminUser(userService.getSimpleUserById(interview.getAdmin()));
             interview.setTechSpecialistUser(userService.getSimpleUserById(interview.getTechSpecialist()));
         });
+        log.info("Return interview with users");
         return interviewWithUserNameDTOList;
     }
 
+    public InterviewWithUserNameDTO getInterviewWithUser(InterviewWithUserNameDTO interviewDTO){
+        MDCLog.putClassNameInMDC(SIMPLE_CLASS_NAME);
+        log.info("Add users to Interview");
+        if (interviewDTO.getAdmin() != null) {
```

Comment:
Why is it not populated by default?

Reply:
Because our interview entity has uuid and has not object references

Reply:
Fixed